src/robot/model/tags.py

```diff
@@ -14,7 +14,7 @@
 #  limitations under the License.
 
 from robot.utils import (Matcher, NormalizedDict, is_string, py2to3, setter,
-                         unic)
+                         unic, normalize)
```

Comment:
We try to keep imports organize alphabetically. This should be placed after `Matcher`. (Although the convention how to sort items in different case isn't that clear in our code base.)

Reply:
Maybe Black would be useful here? https://pypi.org/project/black/